packages/react-app/src/components/quest.tsx

```diff
@@ -257,21 +260,39 @@ export default function Quest({
       <NoPaddingSplitStyled
         primary={
           <Outset gu16 className="pb-0">
+            {isEdit && <TitleStyled>Create Quest</TitleStyled>}
```

Comment:
ModalBase has already a title props for that, have a look at FundModal for example